include/chopper/layout/hierarchical_binning.hpp

```diff
@@ -487,7 +487,7 @@ class hierarchical_binning
         if (data->stats)
         {
             uint64_t const cardinality =
-                config.estimate_union ? data->sketch_toolbox.estimate_interval(trace_j + 1, j) : kmer_count;
+                (!config.disable_estimate_union) ? data->sketch_toolbox.estimate_interval(trace_j + 1, j) : kmer_count;
```

Comment:
```suggestion
                config.disable_estimate_union ? kmer_count : data->sketch_toolbox.estimate_interval(trace_j + 1, j);
```

Seems easier to read than negation?